src/evolution/mutation.py

```diff
@@ -1,6 +1,6 @@
 import random
-def mutation(config, value):
-    if random.random() <= config.mutation_rate:
-        return value + random.uniform(config.mutation_max_difference, -config.mutation_max_difference)
+def mutation(inputs, value):
+    if random.random() <= inputs["chance"]:
```

Comment:
Shouldn't it be called "probability" instead of "chance"?

Reply:
good point